src/main/kotlin/no/nav/medlemskap/regler/v1/arbeidsforhold/JobberBrukerPaaNorskSkipRegel.kt

```diff
@@ -23,7 +24,9 @@ class JobberBrukerPaaNorskSkipRegel(
 
     override fun operasjon(): Resultat {
         return when {
-            arbeidsforhold sisteArbeidsforholdSkipsregister kontrollPeriodeForArbeidsforhold kunInneholder Skipsregister.NOR.name -> ja(regelId)
+            arbeidsforhold sisteArbeidsforholdSkipsregister kontrollPeriodeForArbeidsforhold kunInneholder Skipsregister.NOR.name ||
+                arbeidsforhold.skipsregisterErNISOgFartsområdeErInnenriks(kontrollPeriodeForArbeidsforhold)
```

Comment:
Gjøres om slik at det ikke er noe enten eller. Det vil si at koden også må kunne tillate arbeidsforhold som består av flere arbeidsavtaler hvor det både er NOR skipsregister, og NIS-skipsregister med innenriks fartsområde. Dette kan gjøres ved å legge på logikk i filtreringer i retning: it.skipsregister == NOR || (it.skipsregister == NIS && it.fartsomraade == INNENRIKS)